src/styles/index.scss

```diff
@@ -2,2 +1,107 @@
+// .scss
+$primary-color: #00acdc;
+$font-color: #060b35;
+$light-grey: #616070;
+$font-family: 'Roboto', sans-serif;
+
+html {
   margin: 0;
+  font-family: $font-family;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  width: 200px;
+}
+
+.imac {
```

Comment:
Class names should not be tied to tag names (like '.imac'). Use a class name that describes the purpose or content of the element instead.